public/v2/apps/syncthing.json

```diff
@@ -0,0 +1,43 @@
+{
+   "captainVersion": "2",
+   "documentation": "https://hub.docker.com/r/syncthing/syncthing",
+   "displayName": "Syncthing",
+   "description": "Open Source Continuous File Synchronization.",
+   "dockerCompose": {
+      "services": {
+         "$$cap_appname": {
+            "containerHttpPort": "8384",
+            "ports": [
+               "$$cap_port_sync:9987"
```

Comment:
Where is `9987` coming from? According to their doc the container port is `22000`... Is this working?

Reply:
Thank you for finding this, 22000 is the correct port. 9987 comes from another app definition that I used as a template. I didn't notice I forgot to update the port because in my setup, the application started and synchronized files with remote devices anyway.